### DISCUSSION TITLE
Grid Filtering Question

### DISCUSSION THREAD

**Discussion by krkrueger** [gmtsar/gmtsar#758] [Q&A] [answered]
Hey All,

Quick question regarding simple .grd filtering.  I have some interferograms and aps grids that I was trying to see if I could filter out the deformation signals (they are large spikes but spatially small, about 500m diameter). I thought I could use the grdfilter tool in order to eliminate these. What I tried was:

 gmt grdfilter aps_T1.grd -Gaps_T1test.grd -Fm1 -D4

my understanding from the help was that this would take my grid, perform a median filter with a spatial radius of 1km around each point, and then give me the output grd that has been median filtered.  Nothing changes with my grd.  I have changed the argument on the -Fm from .1 to 10000 and the answer is always exactly the same with no apparent change.  I even tried -Fl with the same radius values without any change either.  What am I missing here?

Thanks in advance!

**Comment by krkrueger** [comment, marked as answer]
Important to be in ll and not radar coordinates...

**Reply by Xiaohua-Eric-Xu** [reply]
Could it be -Fm1k instead?

**Reply by Xiaohua-Eric-Xu** [reply]
In radar coordinates, please use -Dp -Fmx/y, wich x and y being the width of your filter in range and azimuth directions.

**Reply by krkrueger** [reply]
Thanks!

**Comment by krkrueger** [comment]
> In radar coordinates, please use -Dp -Fmx/y, wich x and y being the width of your filter in range and azimuth directions.

So I tried this -Fm11/1 or -Fm11/111 and they give the same answer. No matter what I change the y value to, everything stays the same. What am I missing there?

**Reply by Xiaohua-Eric-Xu** [reply]
Could you provide the full command you used?
